CHANGELOG.rst

```diff
@@ -16,12 +16,74 @@ output of ``git log``
 
    0[1011] machine:dartGIT % git log > full_git_log.txt
 
-A reminder that since many files were moved or renamed, the best way to
+A reminder that since many files were moved or renamed, the best way to 
 get the complete log is to use ``git log --follow`` for information on
 individual files.
 
 The changes are now listed with the most recent at the top.
 
+**April 26 2021 :: preprocess, inflation options, external FO output, 
+wrf-hydro, AMSU-A, DART_LAB Tag: v9.10.0**
+
+*New features*
+
+  -  Updated ``preprocess``:
```

Comment:
these sections (items starting with dashes) are indented one level which makes it look strange when formatted.

Reply:
I agree with Nancy, the formatting looks odd ... must fix before merging.

Reply:
I believe (?) I fixed the formatting. will review in person.

Reply:
yup ... looks good now.